opennms-webapp-rest/src/test/java/org/opennms/web/rest/v2/MonitoringLocationRestServiceIT.java

```diff
@@ -164,4 +172,74 @@ public void testEventOnCreationAndDeletion() throws Exception {
         // delete the one with polling packages
         sendRequest(DELETE, "/monitoringLocations/location2", 204);
     }
+
+    @Test
+    @Transactional
+    public void testLocationLimits() throws Exception {
+
+        final Integer DEFAULT_LIMIT = 10;
+        final Integer LOCATION_COUNT = 15;
+        final ObjectMapper MAPPER = new ObjectMapper();
+
+        this.eventIpcManager.getEventAnticipator().reset();
+
+        // There is one default location
+        LOG.info("Adding locations...");
+        for (int i = 0; i < LOCATION_COUNT - 1; i++) {
```

Comment:
Please explain why need the `-1` here in form of a comment.

Reply:
done